### PR TITLE
Add support for sendpushnotification

### PR DESCRIPTION
**Pull request by bofftompa**
**Why**
After using the tibber-api for a private project I wanted to send a push notification to my TIbber app as part. Since the tibber-api package was missing this I decided to implement support for it.

**How to test:** 
- Change the tibber apikey to your own in the /examples/push-notification.ts and run ts-node push-notification.ts
- The added test 'TibberQuery.sendPushNotification() should return an error when using demo user' is creating a call to Tibber using the demo token, so the test is asserting that the error received is 'operation not allowed for demo user'

### CODE REVIEW

File: src/gql/sendPushNotification.gql.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const cqlSendPushNoticifation = `
```
Comment (bisand) [inline src/gql/sendPushNotification.gql.ts:1]: Please correct the typo. I suspect it should be gqlSendPushNoticifation :)

**Review by bisand — CHANGES_REQUESTED** [review]
Thank you for the contribution! It looks good. Great work!

I have only one comment before approving and merging the PR. This is just a little typo. Rename cqlSendPushNoticifation to gqlSendPushNoticifation and we should be good to go.

Thanks!

File: src/gql/sendPushNotification.gql.ts (hunk 1 above)
Comment (bofftompa) [reply src/gql/sendPushNotification.gql.ts:1]: Fixed in my latest commit

**Comment by bofftompa** [comment]
Thanks! 

I added README documentation to my latest commit that took care of the typo.

**Comment by bisand** [comment]
I actually meant the declaration **cqlSendPushNoticifation** in the file **src/gql/sendPushNotification.gql.ts** it must be corrected to **gqlSendPushNotification**. I'm sorry to be picky, but it's referenced in a couple of places, and the build also fails :)

**Comment by bofftompa** [comment]
Of course! Sorry, I missed adding that change in my previous commit

**Comment by bofftompa** [comment]
Sorry for the typo mess. my autosave did not work so tibberquery.ts was not committed last time 🙁 hopefully the builds passes this time

**Comment by bofftompa** [comment]
Thanks for the awesome package :) 
I can now control the temperature of my Ebeco underfloor heating system based on the current pricelevel :)

**Pull request merged**